src/nccl_ofi_api.cpp

```diff
@@ -237,12 +237,10 @@ ncclResult_t nccl_net_ofi_listen(int dev_id, void *handle, void **lComm)
 		return check_return(ncclInternalError);
 	}
 
-	ret = base_ep->listen(base_ep,
-						  (nccl_net_ofi_conn_handle_t *)handle,
-						  listen_comm);
+	ret = base_ep->listen((nccl_net_ofi_conn_handle_t *)handle, listen_comm);
```

Comment:
we should get rid of the base_* naming here.  It's just the ep.

Also, a static cast should (I think) be sufficient here?

Reply:
In the latest force push I renamed all references to `base_ep` to just be `ep`, including in the sendrecv and rdma transport code. 

I now realize there could be an argument in keeping the `base_ep` naming for `nccl_net_ofi_ep_t` type pointers in the sendrecv and rdma transport code to more clearly convey whether transport-specific member variables/functions are directly accessible by any given endpoint pointer. If this argument is valid I can refactor the naming to keep the `base_*` variable naming in the transport source code and remove `base_*` elsewhere.